core/common/boot/src/main/java/org/eclipse/edc/boot/system/ExtensionLoader.java

```diff
@@ -112,6 +112,21 @@ private static Supplier<Object> getDefaultProviderInvoker(ServiceExtensionContex
         return openTelemetries.isEmpty() ? GlobalOpenTelemetry.get() : openTelemetries.get(0);
     }
 
+    /**
+     * Parses the ConsoleMonitor log level from the program args. If no log level is provided, defaults to Level default.
+     */
+    private static ConsoleMonitor.Level parseLogLevel(String[] programArgs) {
+        return Set.of(programArgs).stream().filter(arg -> arg.startsWith(ConsoleMonitor.LEVEL_PROG_ARG)).map(arg -> getLogLevel(arg.split("=")[1])).findFirst().orElse(ConsoleMonitor.Level.getDefaultLevel());
+    }
+
+    private static ConsoleMonitor.Level getLogLevel(String level) {
+        try {
+            return ConsoleMonitor.Level.valueOf(level);
+        } catch (IllegalArgumentException e) {
+            return ConsoleMonitor.Level.getDefaultLevel();
```

Comment:
I think the runtime should not be started when the log level specified is not valid and the available log levels should be listed.

Reply:
[638be6a](https://github.com/eclipse-edc/Connector/pull/4476/commits/638be6acf4ced6b1909be5e3c50988cbedc7f2a0)